src/kirby/components/list/list-cell-line/list-cell-line.component.tns.html

```diff
@@ -1 +1 @@
-<Label [text]="text" [ngClass]="{primary: primary}" *ngIf="text"></Label>
+<Label [text]="text" [ngClass]="{primary: primary, selected: selected}" *ngIf="text"></Label>
```

Comment:
Be aware of potential performance issue with `[ngClass]`. I suggest to use the `[class.primary]`notation instead.